x-pack/plugin/esql/qa/server/single-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/PushQueriesIT.java

```diff
@@ -74,16 +74,16 @@ public void testEquality() throws IOException {
             """;
         String luceneQuery = switch (type) {
             case "text", "auto" -> "#test.keyword:%value -_ignored:test.keyword";
-            case "match_only_text" -> "*:*";
+            case "constant_keyword", "match_only_text" -> "*:*";
             case "semantic_text" -> "FieldExistsQuery [field=_primary_term]";
             default -> throw new UnsupportedOperationException("unknown type [" + type + "]");
         };
-        boolean filterInCompute = switch (type) {
-            case "text", "auto" -> false;
-            case "match_only_text", "semantic_text" -> true;
+        ComputeSignature dataNodeSignature = switch (type) {
+            case "auto", "constant_keyword", "text" -> ComputeSignature.FILTER_IN_QUERY;
+            case "match_only_text", "semantic_text" -> ComputeSignature.FILTER_IN_COMPUTE;
```

Comment:
Because I was changing a bunch of stuff I put these in alphabetical order while I went. We're going to keep adding stuff to this in the next few months, I think. So sorted order helps.